### PR TITLE
[Feat] Add full terraform.tfvars variable list

### PR DESCRIPTION
**Pull request by evberrypi**
Based on earlier discussion, users would find it helpful to have the full list of values prepopulated (and commented out) in the terraform.tfvars file

These values were autogenerated by running the [terraform-docs](https://terraform-docs.io/) tool
```
terraform-docs tfvars hcl 
```

### CODE REVIEW

**Comment by ikatz-nvidia** [comment]
Missing newlines at the ends of the files, otherwise LGTM

File: aks/examples/cnpack/terraform.tfvars [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
-# prometheus-name = "cnpack-prometheus"
+# admin_group_object_ids                = ""
+# az_monitor-user-managed-id            = "tf-holoscan-identity"
+# azure_log_analytics_retention_in_days = 30
+# azure_log_analytics_sku               = "PerGB2018"
+# cluster_name                          = ""
+# fluentbit-workspace-name              = ""
+# fluentbit_enabled                     = true
+# location                              = ""
+# prometheus-name                       = ""
```
Comment (MaggieXJZhang) [inline aks/examples/cnpack/terraform.tfvars]: Any chance we can fix the naming convention in this MR? Most of them have _ as separator, but this one and " az_monitor-user-managed-id" have "-" as separator

Comment (evberrypi) [reply aks/examples/cnpack/terraform.tfvars]: Done -- fixed both

**Review by ikatz-nvidia — APPROVED** [review]
This looks good.  Am I wrong in observing that it appears to cause a breaking change and would require a major version bump when released?

File: gke/terraform.tfvars [hunk 2]
```diff
@@ -1,8 +1,26 @@
...
-# Do not commit this file to Git if you add sensitive values
-# project_id = ""
-# cluster_name = ""
-# region     = "us-west1"
-# node_zones = ["us-west1-b"]
\ No newline at end of file
+# Sample tfvars file. Uncomment out values to use
+# Do not commit this file to Git with sensitive values
+
+# cluster_name                = ""
```
Comment (sunil1511) [inline gke/terraform.tfvars:7]: To maintain consistency with other tfvars, can you add the cluster_name to be `holoscan-cluster`

File: eks/terraform.tfvars [hunk 3]
```diff
@@ -2,8 +2,53 @@
 # SPDX-License-Identifier: Apache-2.0
 
 # Sample tfvars file. Uncomment out values to use
+# Do not commit this file to Git with sensitive values
+
 # cluster_name = "holoscan-cluster"
 # region  = "us-west-2"
 
 # Optional: If deploying into an existing VPC, use the following variable
```
Comment (sunil1511) [inline eks/terraform.tfvars:10]: Probably this comment should be moved just above `existing_vpc_details` variable.

File: eks/terraform.tfvars [hunk 4]
```diff
@@ -2,8 +2,53 @@
...
-# existing_vpc_details = {vpc_id = "", subnet_ids = ["", ""]}
+# additional_node_security_groups_rules = {}
+# additional_security_group_ids         = []
+# additional_user_data                  = ""
+# aws_profile                           = "development"
+# cidr_block                            = "10.0.0.0/16"
+# cluster_name                          = ""
```
Comment (sunil1511) [inline eks/terraform.tfvars:16]: Already have cluster_name above

File: aks/examples/cnpack/README.md [hunk 5]
```diff
@@ -76,14 +76,14 @@ If it is needed to get the value of `log_analytics_workspace_primary_shared_key`
...
 |------|-------------|------|---------|:--------:|
 | <a name="input_admin_group_object_ids"></a> [admin\_group\_object\_ids](#input\_admin\_group\_object\_ids) | (Required) A list of Object IDs (GUIDs) of Azure Active Directory Groups which should have Owner Role on the Cluster. <br>  This is not the email address of the group, the GUID can be found in the Azure panel by searching for the AD Group<br>  NOTE: You will need Azure "Owner" role (not "Contributor") to attach an AD role to the Kubernetes cluster. | `list(any)` | n/a | yes |
-| <a name="input_az_monitor-user-managed-id"></a> [az\_monitor-user-managed-id](#input\_az\_monitor-user-managed-id) | The user managed identity to *create* for use with the Azure monitor-- at this time this does not accept existing user or system managed identity | `string` | `"tf-holoscan-identity"` | no |
+| <a name="input_az_monitor_user_managed_id"></a> [az\_monitor-user-managed-id](#input\_az\_monitor-user-managed-id) | The user managed identity to *create* for use with the Azure monitor-- at this time this does not accept existing user or system managed identity | `string` | `"tf-holoscan-identity"` | no |
```
Comment (sunil1511) [inline aks/examples/cnpack/README.md:79]: Needs text update `az\_monitor-user-managed-id`

File: eks/terraform.tfvars (hunk 4 above)
Comment (evberrypi) [reply eks/terraform.tfvars:16]: Fixed in !11

Comment (evberrypi) [reply eks/terraform.tfvars:16]: Caught this after the merge. Fixed in #11

File: eks/terraform.tfvars (hunk 3 above)
Comment (evberrypi) [reply eks/terraform.tfvars:10]: Fixed in #11

File: gke/terraform.tfvars (hunk 2 above)
Comment (evberrypi) [reply gke/terraform.tfvars:7]: These are auto generated. The values which appear here are the defaults from `variables.tf`. The tfvars make it simple for people to select the values themselves

**Comment by evberrypi** [comment]
> This looks good. Am I wrong in observing that it appears to cause a breaking change and would require a major version bump when released?

There is no functional change here. The tfvars file is just a helper file. See [Hashicorp's docs](https://github.com/NVIDIA/nvidia-terraform-modules/pull/11/):

> To set lots of variables, it is more convenient to specify their values in a variable definitions file 

This file just has a commented out version of all possible variables

File: aks/examples/cnpack/README.md (hunk 5 above)
Comment (evberrypi) [reply aks/examples/cnpack/README.md:79]: Thanks for catching. Fixed in #11

**Pull request merged**